lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -91,10 +91,9 @@ private function build($data) {
     $this->campaign = Campaign::get($data->nid);
```

Comment:
yeah this is the line that breaks the results, this Campaign::get might need to check that the node exists and is of type campaign. cause we probably shouldn't be allowing signups on other node types, but apparently it has happened.